src/Yesmail/Yesmail.php

```diff
@@ -354,14 +354,30 @@ public function Master_Get($masterId) {
         return $ret;
     }
 
+/* Get an existing Master by name
+*
+* @param int $masterName The name of the master to retrieve
+* @return mixed A JSON decoded PHP variable representing the HTTP response, or false if the master is not found
+* @access public
+*/
+    public function Master_Get_By_Name($masterName) {
+        $ret = Yesmail_MasterId_Get_By_Name($masterName);
+
+        if ($ret !== false) {
+            $ret = $this->Master_Get((int)$ret);
+        }
+
+        return $ret;
+    }
+
     /**
-     * Get an existing Master by name
+     * Get an existing MasterId by name
      *
      * @param int $masterName The name of the master to retrieve
-     * @return mixed A JSON decoded PHP variable representing the HTTP response, or false if the master is not found
+     * @return mixed an int if masterId exist, or false if the master is not found
      * @access public
      */
-    public function Master_Get_By_Name($masterName) {
+    public function Yesmail_MasterId_Get_By_Name($masterName) {
```

Comment:
Ick.   Doesn't this mean the long form of this is \Yesmail\Yesmail_MasterId_Get_By_Name() ?
I think you should follow the pattern and drop the Yesmail_